src/components/common/TopNavbar.tsx

```diff
@@ -5,10 +5,96 @@ import { useCallback } from 'react'
 
 interface TopNavbarProps {
   isAdmin: boolean
+  isApplicant: boolean
   image: string
 }
 
-const TopNavbar: React.FC<TopNavbarProps> = ({ isAdmin, image }) => {
+// Admin Users: Users, Events, Calendar, Recruitment
+const admin = [
+  {
+    href: '/',
+    title: 'Home',
+  },
+  {
+    href: '/users',
+    title: 'Users',
+  },
+  {
+    href: '/events',
+    title: 'Events',
+  },
+  {
+    href: '/calendar',
+    title: 'Calendar',
+  },
+  {
+    href: '/recruitment',
+    title: 'Recruitment',
+  },
+]
+
+const applicants = [
+  {
+    href: '/application-status',
```

Comment:
could you help me to change this to '/status' as i renamed the route in my incoming PR for status page?